internal/uploader.go

```diff
@@ -216,3 +222,40 @@ func (uploader *SplitStreamUploader) Clone() Uploader {
 		blockSize:  uploader.blockSize,
 	}
 }
+
+func (uploader *RegularUploader) ShowRemainingTime() {
+	defer uploader.waitGroup.Done()
+	startTime := time.Now()
+	e := ewma.SimpleEWMA{}
+	e.Add(0)
+	for {
+		totalSizeInt, err := uploader.RawDataSize()
+		if err != nil {
+			return
+		}
+		uploadedSizeInt, err := uploader.UploadedDataSize()
+		if err != nil {
+			return
+		}
+
+		totalSize := float64(totalSizeInt) / 1000000
+		uploadedSize := float64(uploadedSizeInt) / 1000000
+
+		remainingSize := totalSize - uploadedSize
+
+		timeElapsed := time.Since(startTime).Seconds()
+		uploadSpeed := uploadedSize / timeElapsed
+		e.Add(uploadSpeed)
+
+		remainingTime := time.Duration(remainingSize/uploadSpeed) * time.Second
```

Comment:
Why we can't utilize more accurate ewma that we use to calculate the remaining time?